templates/vue/src/components/lightbox/H5PIframe.vue

```diff
@@ -175,36 +174,6 @@ export default {
     handleLoad() {
       this.$emit("is-loaded")
 
-      $("iframe").each(function() {
-        $(this)
-          .data("ratio", this.height / this.width)
-          // Remove the hardcoded width & height attributes
-          .removeAttr("width")
-          .removeAttr("height")
-      })
-      const setIframeDimensions = function() {
-        $("iframe").each(function() {
-          // Get the parent container's width
-          var width = $(this)
-            .parent()
-            .width()
-          var height = $(this)
-            .parent()
-            .height()
-          if (width * $(this).data("ratio") <= height) {
-            $(this)
-              .width(width)
-              .height(width * $(this).data("ratio"))
-          } else {
-            $(this)
-              .height(height)
-              .width(height / $(this).data("ratio"))
-          }
-        })
-      }
-      $(window).resize(setIframeDimensions)
-      setIframeDimensions()
```

Comment:
I removed this code because it was causing the `h5p` video to resize to the size of the subaccordion row, making it too small. This didn't seem to have any visible effects to regular h5p videos.

Reply:
Ok. I tested it a bit as well and it seems to work fine except when you view the H5P video in "fullscreen", the video sometimes gets cut off at the bottom. That might be something we have to address at a different time.